CMakeLists.txt

```diff
@@ -817,7 +838,7 @@ add_library_variants(
     aarch64         ""                  "-march=armv8-a"                                        "-M raspi3b"
     armv4t          ""                  "-march=armv4t"                                         "-M musicpal -cpu arm926"
     armv5te         ""                  "-march=armv5te"                                        "-M musicpal -cpu arm926"
-    armv6m          soft_nofp           "-mfloat-abi=soft -march=armv6m"                        "-M microbit"
+    armv6m          soft_nofp           "-mfloat-abi=soft -march=armv6m"                        "-M mps2-an385,accel=tcg"
```

Comment:
Comment please to explain why `accel` is needed and what `tcg` means

Reply:
Actually accel tcg is not necessary to have tests working so it's removed now.